DFC.App.JobGroups/Controllers/PagesController.cs

```diff
@@ -22,12 +22,14 @@ public class PagesController : Controller
         public const string RegistrationPath = "job-groups";
         public const string LocalPath = "pages";
 
+        private const string ExpiryAppSettings = "Cms:Expiry";
         private readonly ILogger<PagesController> logger;
         private readonly AutoMapper.IMapper mapper;
         private readonly IDocumentService<JobGroupModel> jobGroupDocumentService;
         private readonly ISharedContentRedisInterface sharedContentRedis;
         private readonly IConfiguration configuration;
         private string status;
+        private double expiry = 4;
```

Comment:
Please, can a time unit be used, if this is for time?
Seconds, minutes, hours. As in `expiryTimeInXXX`.